components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/test/java/org/wso2/carbon/apimgt/rest/api/publisher/common/SampleTestObjectCreator.java

```diff
@@ -0,0 +1,285 @@
+/*
```

Comment:
We had to duplicate this since we can't directly use the similar one from the core module? Shouldn't we create some common module for classes like this? (@prasa7 had this idea not sure he discussed about this)